publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+# Results of accessibility self-review for Web of Things (WoT) Architecture
+
+Answers to accessibility (FAST) review checklist [(from here)](https://w3c.github.io/apa/fast/checklist.html)
+for the Web of Things (WoT) architecture as a whole.
+These answers are generally written to be applicable to all WoT deliverables.
+Where a response is specifically relevant to a particular deliverable that will be identified.
+
+## If technology allows visual rendering of content
+The WoT architecture does not directly support visual rendering of content.
+Some devices adhering to WoT standards may have visual display aspects, for example
+light bulbs that can be turned on or off, or small displays.
+However, WoT in general has no provision to constrain visual displays, 
+as this (and other controls limiting the behavior of devices)
+was not in the scope of our current charter.
+
+### [no] There is a defined way for a non-visual rendering to be created.	
+No.
+### [no] Content can be resized.	
+No specified mechanism.
+### [possible] Luminosity and hue contrast can adapt to user requirements.	
+No specified mechanism, although WoT TDs for lights usually have such controls, and WoT supports them.
+### [possible] Text presentation attributes can be changed.
+No specified mechanism, although WoT TDs may support controls.
+### [possible] Visual presentation of pointers and cursors can be adjusted.
+No specified mechanism, although WoT TDs may support controls.
+### [no] Changing content presentation does not render it unreadable.
```

Comment:
suggest n/a

Reply:
for questions asking if something *can* be done I have answered "possible".  For questions asking if we are *preventing* something bad from happening I have said "no".  It's certainly possible for an IoT device to display text that is unreadable for some people.